packages/server/src/utils/helpers.ts

```diff
@@ -2,6 +2,13 @@ import path from "path";
 import slugify from "slugify";
 import { nanoid } from "nanoid";
 
+const sanitizeSpecialChar = (fileKey: string) => {
+  const specialCharactersRegex = /[!@#^&*()"':~]/g;
+  return slugify(fileKey, {
+    remove: specialCharactersRegex,
```

Comment:
I think we cannot use `remove` property to achieve what we want here as it will remove only the given characters, thus it will avoid the default behavior and allow any non-english characters. It might create problems in the future. I believe @abul2285 has discussed this with you.

Reply:
Yes bhai, that's why in this method we are only removing the special characters, after that we are slugifying again to remove non-english characters.

Reply:
Updated the method used string replace. Now any character of the file name that is not in (a-z or A-Z) or a digit (0-9) will be replaced with an empty string.

Reply:
Please add a random name if the entire file name is replaced with empty string.